src/lib/ngsi/UpdateActionType.cpp

```diff
@@ -38,13 +38,7 @@
 *
 * UpdateActionType::check - 
 */
-std::string UpdateActionType::check
```

Comment:
Four parameters not actually needed and making the function call complex. Removing.